ci/container/test_authentication.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -e
+
+set -o pipefail
+
+export AUTH_PARAMETER_FILE=./.github/workflows/parameters_aws_auth_tests.json
+eval $(jq -r '.authtestparams | to_entries | map("export \(.key)=\(.value|tostring)")|.[]' $AUTH_PARAMETER_FILE)
+
+go test -run TestExternalBrowser*
+go test -run TestClientStoreCredentials
+go test -run TestOkta*
+go test -run TestOauth*
```

Comment:
I wonder if there is any better way to not forget about adding a line here, when auth tests are added...

Reply:
Up to you, to me it's fine as it is, possibly:
1. Change name of functions and use just one command.  _go test -run TestAuthWith*_
2 Perhaps create a flag and use it here?
3. Provide certain files, instead of function names?